src/controller.cpp

```diff
@@ -290,28 +290,10 @@ void Controller::handshakeTimeout() {
 }
 
 void Controller::serverUnavailable() {
-  logger.error() << "server unavailable";
+  logger.info() << "Server Unavailable - Ping succeeded: " << m_pingReceived;
 
-  // If VPN is already active and server location becomes unavailable we do not
-  // deactivate the VPN or proceed with next steps. We specifically do not want
-  // to deactivate the VPN in the case of server location becoming unavailable
-  // because the user may not notice that they are no longer protected which can
-  // cause the traffic to leak outside the tunnel without their knowledge.
-  if (m_state == StateOn || m_state == StateOnPartial) {
-    return;
-  }
-
-  m_nextStep = ServerUnavailable;
```

Comment:
Can we can remove `ServerUnavailable` in `controller.h` (around line 193)?

Reply:
... I will confirm and report back. I think so.

Reply:
removed